docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -734,54 +734,130 @@ Baseline coverage of treatment for elevated SBP and elevated LDL-c is substantia
-    - Is change in risk factor by state also important? (i.e., a XX drop in SBP)
-    - What is missing from the below? 
 
 Outputs:
 
-#. DALYs (YLLs and YLDs) due to cause specific mortality 
-#. Average values for risk factors by state and year (SBP, LDL-C, FPG, and BMI)
-#. Numbers of interventions administered per a) 100,000 population, and b) 100,000 person years 
+#. Total population 
+#. Person-time 
+#. YLLs and YLDs
+#. Deaths 
+#. Transitions for each cause 
+#. Exposure by person time for all risk factors 
+#. Person time at or below target values for SBP and LDL-C 
+#. Healthcare appointments 
+#. Person time on medication 
+#. Medication effect 
+#. Numbers of interventions 
+
 
+Stratifications for All: 
+
+#. Year 
+#. Age-group 
+#. Sex 
+#. State (Alabama, Alaska, etc)
+#. Scenario 
+#. Race (note: not included in minimum viable model, to be added later)
 
 .. _uscvd4.8:
 
 4.8 Output meta-table shell
 ---------------------------
 
-.. todo::
-  - add table 
+.. list-table:: Model Outputs 
+  :widths: 5 15 15 
+  :header-rows: 1
+
+  * - Output 
+    - Notes
+    - Additional Stratifications Needed 
+  * - Population  
+    - 
+    -  
+  * - Person-time  
+    - sum of total person time
+    - By state for each cause (i.e., suscepitble vs acute for myocardial infarction)
+  * - YLLs  
+    - sum of YLLs for cause i 
+    - Stratify by cause 
+  * - YLDs  
+    - sum of YLDs for cause i 
+    - Stratify by cause 
+  * - Deaths 
+    - sum of deaths for cause i 
+    - Stratify by cause 
+  * - Transitions between states 
+    - sum of transitions between states within cause i 
+    - i.e., transition from susceptible to acute MI, stratified by cause 
+  * - Mean SBP 
+    - sum of SBP * person time
+    - 
+  * - Mean LDL-C
+    - sum of LDL-C * person time
+    - 
+  * - Mean BMI 
+    - sum of BMI * person time *NOTE: NOT IN CURRENT MODEL*
+    - 
+  * - Mean FPG 
+    - sum of FPG * person time *NOTE: NOT IN CURRENT MODEL*
+    - 
+  * - Population achieving target LDL-C
+    - sum of person time at or below 1.81 LDL-C 
+    - 
+  * - Population achieving target SBP
+    - sum of person time at or below 130 SBP  
+    - 
+  * - Healthcare appointments 
+    - sum of healthcare appointments 
+    - Split by type of appointment - follow-up vs emergency vs screening 
+  * - Population on SBP medication 
+    - sum of person time on SBP medication 
+    - Split by primary non-adherent, secondary non-adherent, and adherent; and split by medication category 
+  * - Population on LDL-C medication 
+    - sum of person time on LDL-C medication 
+    - Split by primary non-adherent, secondary non-adherent, and adherent; and split by medication category 
+  * - SBP medication effect 
+    - sum of change in SBP exposure * person time  
+    - Split by medication category 
+  * - LDL-C medication effect 
+    - sum of change in LDL-C exposure * person time  
+    - Split by medication category 
```

Comment:
Hmm, maybe I don't understand enough about the treatment model here, but my instinct would be to split the mean SBP/LCL outputs by medication status and then calculate the difference from there

Reply:
That makes sense and is aligned to what the engineers said - I adjusted the docs to reflect this change

Reply:
Cool, I think we could actually delete these rows and then add medication to the list of stratifications in the "mean SBP/LCL-C" rows above.

Reply:
I see your point - I updated the docs

Reply:
Cool looks good!